examples/forecasts/example_forecasting_task.py

```diff
@@ -0,0 +1,26 @@
+from forecasting.task.forecasting.forecasting import Forecasting as task
```

Comment:
again all imports should be importing autogluon as well (assume this code is run outside of the autogluon package by users)